docs/UserGuide.md

```diff
@@ -44,136 +37,38 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 ## Features
 
-<box type="info" seamless>
```

Comment:
Since we are sticking with the AB3 format, we should keep the notes about the command format section (with minor edits) to explain the users how to understand our commands.